src/scripts/main.js

```diff
@@ -355,6 +355,21 @@ const people = [
 ];
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dashboard = document.querySelector('.dashboard');
 
-// write your code here
+const visiblePeople = people.map((person) => ({
+  name: person.name,
+  sex: person.sex,
+  born: person.born,
+  died: person.died,
+  age: person.died - person.born,
+  century: Math.ceil(person.died / 100),
+}));
+
+for (let i = 0; i <= visiblePeople.length; ++i) {
```

Comment:
This loop will throw an error when `i` equals `visiblePeople.length`, because `visiblePeople[visiblePeople.length]` is `undefined`. You should use a strict less than comparison operator `<` instead of `<=`, so the loop will iterate while `i` is less than `visiblePeople.length`. This is a common mistake known as an off-by-one error.